eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -387,12 +393,30 @@ function GetExistingTags($apiUrl) {
   }
 }
 
-# Walk across all build artifacts, check them against the appropriate repository, return a list of tags/releases
-function VerifyPackages($pkgRepository, $artifactLocation, $workingDirectory, $apiUrl, $releaseSha,  $continueOnError = $false) {
-  $pkgList = [array]@()
+# Retrieve release tag for artiface package. If multiple packages, then output the first one.
+function RetrieveReleaseTag($pkgRepository, $artifactLocation, $continueOnError = $true) {
+  if (!$artifactLocation) {
+    return ""
```

Comment:
Is this an exceptional condition? If we ever pass in $artifactLocation it should always be present right?

Reply:
This is a helper function. When we checked in the code, there is no actual $artifactLocation passed in. We have to make changes to each lang release yml. If throw exception here, then we cannot guarantee the backward compatiblity.

Reply:
Yes it is about making sure we don't break existing pipelines.